### PR TITLE
SPARCRequest navigation changes

### PR DESCRIPTION
**Pull request by ying-pbrc**
Please review and let me know if you have any comments/suggestions.
Thanks,
Ying

### CODE REVIEW

File: config/application.yml.example [hunk 1]
```diff
@@ -58,11 +58,6 @@ development:
   system_satisfaction_survey: false
   editable_statuses: { 14: ['first_draft', 'draft', 'submitted', nil, 'get_a_cost_estimate', 'awaiting_pi_approval'],
                      48: ['draft', 'submitted'] }
```
Comment (jleonardw9) [inline config/application.yml.example]: I think we need to leave the navbar links configurable in the yml file. Some institutions may not want a fulfillment link and the SPARCAdmin link will be going away when we go live with our bootstrap branch. It also makes it easy to point these different places for testing.

File: app/views/shared/_navbar.html.haml [hunk 2]
```diff
@@ -1,7 +1,20 @@
 %ul.navbar
```
Comment (jleonardw9) [inline app/views/shared/_navbar.html.haml]: See comment below.

File: config/locales/en.yml [hunk 3]
```diff
@@ -35,7 +35,7 @@ en:
   feedback: "Feedback"
   issue_contact: "Please contact the SUCCESS Center at (843) 792-8300 or success@musc.edu for assistance with this process or with any questions you may have."
   portal_title: "SPARC Request Portal"
```
Comment (jleonardw9) [inline config/locales/en.yml]: Please do not make changes to this file, we need to maintain out own configuration. If you look in the readme, we show you how to set your own file up for localization.

Comment (ying-pbrc) [inline config/locales/en.yml]: I can undo the change. However, the original text is too long, it doesn't fit the button - see attached screenshot. Any other suggestions?
create an account

File: config/application.yml.example (hunk 1 above)
Comment (ying-pbrc) [inline config/application.yml.example]: I wasn't aware of the new Dashboard change when this pull request was submitted. With that in mind, let me rework on this.

File: config/locales/en.yml (hunk 3 above)
Comment (jleonardw9) [inline config/locales/en.yml]: Sorry, you're right. I had the button turned off locally and didn't see the problem. Looks good!

File: config/application.yml.example (hunk 1 above)
Comment (ying-pbrc) [inline config/application.yml.example]: I moved the navigation links configurable from constant.yml to application.yml; modified the navbar partial and the index view in catalog_manager/catalog. I changed the name from navbar_links to nav_links as they are not limited to the navbar usage.

**Comment by ying-pbrc** [comment]
After seeing the new dashboard change in v.1.7.0, I believe this pull request is no longer necessary.

**Pull request closed without merging**